code/game/machinery/requests_console.dm

```diff
@@ -193,7 +193,7 @@ GLOBAL_LIST_EMPTY(allRequestConsoles)
 					reset_message(TRUE)
 
 		if("writeAnnouncement")
-			var/new_message = sanitize(input("Write your message:", "Awaiting Input", ""))
```

Comment:
Make sure you cant XSS with this please.

Reply:
Both of them still get sanitised in the `Announce()` proc here:
https://github.com/ParadiseSS13/Paradise/blob/156d360e57bd57516e934e841e3301d741a5036f/code/defines/procs/announce.dm#L54-L56
As far as I can tell it's all working the same as before.